### ISSUE TITLE
Some charts not meeting accessibility

### ISSUE THREAD

**Issue by heatherco** [carbon-design-system/carbon#5036]
Hi, Unless these chart are always accompanied by a data table, they do not meet accessibility.

Below are the charts that do not meet accessibility and some suggestions of what can be done to address the issues.

**Provide values for each bar:**
<img width="1157" alt="image" src="https://user-images.githubusercontent.com/12818623/72366635-a5b9d580-36c8-11ea-93b6-62e830ca91e4.png">

**Provide values inside or beside bars:**
<img width="369" alt="image" src="https://user-images.githubusercontent.com/12818623/72366747-de59af00-36c8-11ea-983c-e9be35f29423.png">

**Provide each item a different shape to represent data point or same shape but different line styling:**
<img width="1035" alt="image" src="https://user-images.githubusercontent.com/12818623/72366857-17921f00-36c9-11ea-8d4d-306f787bbce5.png">

Like this for different shapes:
<img width="582" alt="image" src="https://user-images.githubusercontent.com/12818623/72367146-ad2dae80-36c9-11ea-91a2-e6c6b67763a1.png">

Like this for different line styles:
<img width="444" alt="image" src="https://user-images.githubusercontent.com/12818623/72367262-ec5bff80-36c9-11ea-816b-8dd64de1d2f0.png">

<img width="534" alt="image" src="https://user-images.githubusercontent.com/12818623/72367224-d3ebe500-36c9-11ea-9c48-a8192f2b2276.png">

**Legend does not appear in the correct order.** The order does not reflect the order in the charts
<img width="911" alt="image" src="https://user-images.githubusercontent.com/12818623/72368353-e8c97800-36cb-11ea-8b58-6e37bacec814.png">

**Scatter charts.** 
1) The colors in the legend does not match what is on the graph
<img width="1149" alt="image" src="https://user-images.githubusercontent.com/12818623/72368602-65f4ed00-36cc-11ea-902e-8ae8f20fd8d9.png">

 2) Different shapes should be used to distinguish between each item:

<img width="522" alt="image" src="https://user-images.githubusercontent.com/12818623/72368817-c7b55700-36cc-11ea-885e-7f02bdf8d8b6.png">

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/12818623/72368877-e582bc00-36cc-11ea-8e23-ffc7e4f8068a.png">

**Comment by dakahn** [comment]
Excellent feedback! ❤️ 

@jeanservaas I'm assuming our guidance for complex charts is to provide an accessible alternative way to access the information (like was mentioned in the post, a data table is a good solution for smaller data sets especially) until we can build in/design more robust and accessible charts.

**Comment by asudoh** [comment]
Hi @heatherco do you want to report this to https://github.com/carbon-design-system/carbon-charts? Thanks!

**Comment by theiliad** [comment]
Hi @heatherco, thanks for bringing these points up. Some of these were noticed by design, however there was never a resolution that was approved to be implemented. I'll bring this up again with design.

In the meantime I suggest closing the issue here and copying the content over to https://github.com/carbon-design-system/carbon-charts/issues/467

**Comment by asudoh** [comment]
Moved to: #467
